src/cpuprofiler.js

```diff
@@ -15,13 +15,17 @@
 // That doesn't work for Chrome in turn, so need to resort to user agent
 // sniffing.. (sad :/)
 if (!performance.realNow) {
-  var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
-  if (isSafari) {
-    var realPerformance = performance;
-    performance = {
-      realNow: () => realPerformance.now(),
-      now: () => realPerformance.now()
-    };
+  if (typeof navigator !== "undefined") {
+    var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
```

Comment:
Can you instead roll this into the existing `isSafari`?  (to avoid adding the extra `else`)?

e.g. ` var isSafari =  typeof navigator !== "undefined" && /^((?!chrome|android).)*safari/i.test(navigator.userAgent);`